<ui version="4.0" >
 <class>AddVideo</class>
 <widget class="QDialog" name="AddVideo" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>97</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Add Video URL</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/buttons/images/film_add.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lblTitle" >
     <property name="text" >
      <string>Enter the video URL to download:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="edtURL" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="spbPasteURL" >
     <property name="toolTip" >
      <string>Paste URL from clipboard</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/buttons/images/paste_plain.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="imgService" >
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblVideoService" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>181</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddVideo</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>491</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>490</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
